dashboard/app/models/ability.rb

```diff
@@ -69,7 +69,7 @@ def initialize(user)
     can [:show, :index], DataDoc
 
     # If you can see a level, you can also do these things:
-    can [:embed_level, :get_rubric, :get_serialized_maze, :level_data], Level do |level|
+    can [:embed_level, :get_rubric, :get_serialized_maze, :level_properties], Level do |level|
```

Comment:
Slightly unrelated, but I had forgotten to update this when changing the level_data route, and this was causing standalone projects to fail for non-levelbuilders.